packages/expo-doctor/src/utils/explainDependencies.ts

```diff
@@ -0,0 +1,132 @@
+import spawnAsync, { SpawnResult } from '@expo/spawn-async';
+import chalk from 'chalk';
+import semver from 'semver';
+
+import { RootNodePackage, VersionSpec } from './explainDependencies.types';
+
+type TargetPackage = { name: string; version?: VersionSpec };
+
+function isSpawnResult(result: any): result is SpawnResult {
+  return 'stderr' in result && 'stdout' in result && 'status' in result;
+}
+
+/** Spawn `npm explain [name] --json` and return the parsed JSON. Returns `null` if the requested package is not installed. */
+async function explainAsync(
+  packageName: string,
+  projectRoot: string,
+  parameters: string[] = []
+): Promise<RootNodePackage[] | null> {
+  const args = ['explain', packageName, ...parameters, '--json'];
+
+  try {
+    const { stdout } = await spawnAsync('npm', args, {
+      stdio: 'pipe',
+      cwd: projectRoot,
+    });
+
+    return JSON.parse(stdout);
+  } catch (error: any) {
+    if (isSpawnResult(error)) {
+      if (error.stderr.match(/npm ERR! No dependencies found matching/)) {
```

Comment:
If you use `--json`, why not use the actual json output, instead of parsing human readable output.
```
❯ npm explain bla --json
npm ERR! No dependencies found matching bla
{
  "error": {
    "code": null,
    "summary": "No dependencies found matching bla",
    "detail": ""
  }
}
```